enclave-runner/src/usercalls/mod.rs

```diff
@@ -148,7 +124,7 @@ impl Read for Stdin {
         }
 
         lazy_static::lazy_static! {
-            static ref STDIN: Lock<AsyncStdin> = {
+            static ref STDIN: futures::lock::Mutex<AsyncStdin> = {
```

Comment:
We need to use `futures::lock::Mutex` in some case because it allows calling `lock()` for `T: ?Sized` while tokio's Mutex does not. I replaced the tokio Mutex use with futures one